core-rust/state-manager/src/store/db.rs

```diff
@@ -338,3 +344,114 @@ impl RecoverableVertexStore for StateManagerDatabase {
         }
     }
 }
+
+pub enum WriteableStateManagerDatabaseAccountChangeIndex<'a> {
```

Comment:
I'm a little surprised by this - why isn't this part of the StateManagerDatabase?